doc/snippets/Microsoft.Data.SqlClient/ActiveDirectoryAuthenticationProvider.xml

```diff
@@ -40,6 +40,9 @@
       <summary>Acquires a security token from the authority.</summary>
       <returns>Represents an asynchronous operation that returns the authentication token.</returns>
     </AcquireTokenAsync>
+    <ClearUserTokenCache>
+      <summary>Clears cached instances of Public Client Application maintained by driver to access cached tokens from underlying token provider library.</summary>
```

Comment:
I think we can keep the description simple:

```suggestion
      <summary>Clears cached user tokens from the token provider.</summary>
      <remarks>This will cause interactive authentication prompts to appear again if tokens were previously being obtained from the cache.</remarks>
```